src/commands/__tests__/extractThought.ts

```diff
@@ -48,18 +54,24 @@ describe.skip('Extract thought', () => {
   })
 
   it('the selected part of a thought is extracted as a child thought', async () => {
+    initialize()
+    await timer.runAllAsync()
```

Comment:
Are you sure this is needed? `initialize` is already [called in createTestApp](https://github.com/cybersemics/em/blob/e97343c9a4e1daae0690c21fe212b205f6333607/src/test-helpers/createTestApp.tsx#L20) before each test.

Reply:
Now I removed the initialize from start of the test cases.
I added this to avoid the errors "_triggering the "Missing docKey for thought_" error", I was facing during the execution of the test cases. Now I handled differently and cleaned the code, and all removed unnecessary wait times.